src/pages/concept-list-page/concept-list/components/concept-list-collapse-item/index.tsx

```diff
@@ -39,6 +49,11 @@ const CollapseItem: FC<Props> = ({ concepts }) => {
           {highestPublishedVersionConcept?.versjonsnr?.minor}.
           {highestPublishedVersionConcept?.versjonsnr?.patch}
         </SC.Column>
+        <SC.Column>
+          {newestChange !== Infinity
+            ? DateTime.fromMillis(newestChange).toLocaleString()
```

Comment:
Ser at din dato vises på feil format i staging. Tror DateTime.fromMillis(newestChange).toFormat('DD.MM.YYYY'); vil fungere.

Reply:
Eller kanskje man skal tillate hvis bruker har engelsk språk i browser så ser man det dato-formatet?

Reply:
Ah, vises på norsk format hos meg. Skal jeg bare sette norsk locale for alle?